test/models/pentest_flag_test.rb

```diff
@@ -28,6 +32,12 @@ def setup
     assert_equal 50, point_value
   end
 
+  test 'defense challenge sponsorship' do
```

Comment:
Could you also add a test to verify that this data correctly displays on all the pages it is supposed to display on? (see the game integration tests for examples).